modules/ROOT/partials/configuration/extended_valid_elements.adoc

```diff
@@ -9,7 +9,6 @@ The below example replaces the current `img` rule (including the global element
 
```

Comment:
In the paragraphs above there are some refs that link to things in other files, it's better to include their full refs like so :
Line 4 and 6: ```xref:valid_elements[`valid_elements`]``` needs to include the full ref
Line 4:  ```xref:schema[`schema`]``` needs to include the full ref